pkgs/shells/lambda-mod-zsh-theme/default.nix

```diff
@@ -2,15 +2,15 @@
 
 let
   repo = "lambda-mod-zsh-theme";
-  rev = "eceee68cf46bba9f7f42887c2128b48e8861e31b";
+  rev = "c6445c79cbc73b85cc18871c216fb28ddc8b3d96";
 in stdenv.mkDerivation {
   name = "${repo}-${rev}";
```

Comment:
not sure if I got it right: you mean `year-mongth-day` of the latest commit included in our derivation? (same way as implemented in oh-my-zsh pkg?)